src/server/index.js

```diff
@@ -63,8 +63,10 @@ const port = process.env.DEV_APP_PORT || process.env.PORT
 
 // Don't rate limit heroku
 app.enable('trust proxy')
-if (!DEBUG && process.env.PUBLIC_DIR) {
-  app.use(express.static(process.env.PUBLIC_DIR, {
+
+// Serve static assets
+if (existsSync(process.env.ASSETS_DIR)) {
```

Comment:
is this an intentional change from PUBLIC_DIR => ASSETS_DIR?
Can you explain why it's ASSETS_DIR and not PUBLIC_DIR?

Also, why are we removing the DEBUG condition? I suppose some containerization deployments have simplicity benefits in serving static assets through node rather than a static file server? If so, I think we should at least have an env var that can disable this (no matter the presence of ASSETS_DIR).

(cc: https://github.com/MoveOnOrg/Spoke/pull/1085#pullrequestreview-222779938)

Reply:
> is this an intentional change from PUBLIC_DIR => ASSETS_DIR?
Can you explain why it's ASSETS_DIR and not PUBLIC_DIR?

Yes, I noticed that `PUBLIC_DIR` only works if it contains `assets`, which is potentially very confusing (if you set `ASSETS_DIR` to something else during build then you'll have to do some digging to figure out why the app doesn't load).

I also could not figure out why specifying `PUBLIC_DIR` as the level above `assets` might be necessary (if you have insight here please let me know), and noticed that `ASSETS_DIR` is also required at runtime:

https://github.com/MoveOnOrg/Spoke/blob/9078ef59606e2608817bdb8a56a8fce5e4992256/src/server/middleware/app-renderer.jsx#L30

So by removing `PUBLIC_DIR`, we only have to set one environment variable and its usage is more declarative.

> Also, why are we removing the DEBUG condition?

I had no idea why we might want to turn off static asset serving when DEBUG is enabled. Is there a use-case for this? Unsetting the `ASSETS_DIR` or removing the `ASSETS_DIR` from path would also disable static file serving.